tests/ttnn/unit_tests/operations/test_clone.py

```diff
@@ -4,17 +4,96 @@
 
 import pytest
 import torch
+import copy
 
 import ttnn
 from models.utility_functions import comp_allclose_and_pcc
 from loguru import logger
 
 from tests.ttnn.unit_tests.operations.test_utils import (
-    to_cpu,
-    to_npu,
+    get_compute_kernel_options,
+    compute_kernel_options,
+    compute_kernel_ids,
 )
 
 
+def to_ttnn(
+    torch_tensor,
+    device,
+    *,
+    ttnn_layout=ttnn.TILE_LAYOUT,
+):
```

Comment:
Instead of using `to_ttnn`, how about modifying `to_npu` so that it doesn't apply padding in the case of `row_major`? When `to_npu` was originally created, supporting `row_major` wasn't considered, so that aspect wasn't taken into account.

Reply:
Yes, I've considered that, but it seems out of scope for our epic. The deadline for Epic MV-842 is approaching quickly, so I'll plan to do that later, over the weekend, after the Epic deadline. Is that ok?

Also, modifying to_npu could affect other operations that I haven't investigated yet.